### PR TITLE
Search: prevent closing of the pop-up on autocomplete select

### PR DESCRIPTION
**Pull request by gu-stav**
## What does this change?

FIxes a bug, where the search pop-up was closed, if somebody clicks on the autocompletion results.

[Trello reference](https://trello.com/c/PbC7chMB/297-search-bug)

## What is the value of this and can you measure success?

Better search.

## Does this affect other platforms - Amp, Apps, etc?

No.

## Screenshots

No.

## Tested in CODE?

No.

### CODE REVIEW

File: static/src/javascripts-legacy/projects/common/modules/navigation/search.js [hunk 1]
```diff
@@ -63,7 +68,6 @@ define([
                 // Make sure search is always in the correct state
                 self.focusSearchField();
                 e.preventDefault();
-                mediator.emit('modules:search');
```
Comment (NataliaLKB) [inline static/src/javascripts-legacy/projects/common/modules/navigation/search.js]: Why are you removing mediator for this change?

Comment (gustavpursche) [inline static/src/javascripts-legacy/projects/common/modules/navigation/search.js]: Because it's not used anywhere :)

Comment (NataliaLKB) [inline static/src/javascripts-legacy/projects/common/modules/navigation/search.js]: Thanks a good reason! Might be good in future to keep that as a separate commit as it doesn't actually relate to `"Search: prevent closing of the pop-up on autocomplete select"`

**Review by NataliaLKB — APPROVED** [review]
lgtm 👍

File: static/src/javascripts-legacy/projects/common/modules/navigation/search.js (hunk 1 above)
Comment (gustavpursche) [inline static/src/javascripts-legacy/projects/common/modules/navigation/search.js]: You are totally right <3. I decided too late, I'd need a refactoring PR like this: https://github.com/guardian/frontend/pull/16831 ;)

**Comment by stephanfowler** [comment]
Nice one :-)

**Pull request merged**